demo/pages/sdk/actions/swap.tsx

```diff
@@ -181,19 +188,42 @@ const SwapActionPage: NextPage = () => {
           cursor: 'pointer'
         }}
         onClick={async () => {
-          if (!wallet) {
+          if (!primaryWallet) {
             throw 'Please connect to execute transactions'
           }
-          if (recipient && !isAddress(recipient)) {
-            throw 'Recipient must be an address'
-          }
+
           if (!amount) {
             throw 'Must include an amount for swapping'
           }
 
+          let executionWallet
+
+          if (fromChainId === 792703809) {
```

Comment:
should we be hardcoding this? I feel like we should just look up the chain's vmType